lms/js_config_types.py

```diff
@@ -130,3 +130,6 @@ class User(TypedDict):
 class DashboardConfig(TypedDict):
     user: User
     routes: DashboardRoutes
+
+    sync_enabled: bool
```

Comment:
Since this is the general config for the dashboard, perhaps `auto_grading_sync_enabled` would be a more descriptive name of what sync is enabled.

Reply:
Yup, that's a better name :+1: